packages/docs/pages/Alert/AlertPage.tsx

```diff
@@ -0,0 +1,148 @@
+import { alertsManager, Alert, AlertsManager, AlertProps, Button, H0, H1, H2, Text } from '@bigcommerce/big-design';
+
+import { Code, CodePreview, CodeSnippet } from '../../components';
+import {
+  AlertsManagerAddMethodList,
+  AlertsManagerRemoveMethodList,
+  AlertsManagerSubscribeMethodList,
+  AlertsManagerUnsubscribeMethodList,
+} from '../../MethodLists';
+import { MessagingItemPropTable, MessagingLinkItemPropTable } from '../../PropTables/shared';
+import { AlertPropTable } from '../../PropTables/AlertPropTable';
+
+const errorAlert = {
+  header: 'Optional Headline',
+  messages: [{ text: 'Required description copy.' }],
+  type: 'error',
+  onClose: () => null,
+} as AlertProps;
+
+const successAlert = {
+  header: 'Optional Headline',
+  messages: [{ text: 'Required description copy.', link: { text: 'Optional Link', href: '#' } }],
+  type: 'success',
+  onClose: () => null,
+} as AlertProps;
+
+const warningAlert = {
+  header: 'Optional Headline',
+  messages: [
+    {
+      text: 'Required description copy.',
+      link: { text: 'External Link', href: '#', external: true, target: '_blank' },
+    },
+  ],
+  type: 'warning',
+  onClose: () => null,
+} as AlertProps;
+
+const infoAlert = {
+  header: 'Optional Headline',
+  messages: [
+    {
+      text: 'Required description copy.',
+      link: {
+        href: '#',
+        text: 'Optional Link',
+      },
+    },
+    {
+      text: 'Required description copy.',
+      link: {
+        text: 'External Link',
+        href: '#',
+        external: true,
+        target: '_blank',
+      },
+    },
+  ],
+  type: 'info',
+  onClose: () => null,
+} as AlertProps;
+
+export default () => (
+  <>
+    <H0>Alerts</H0>
+    <Text>An alert appears at the top right of the interface notifying the user after an action.</Text>
+
+    <CodeSnippet>
```

Comment:
What do you think about having a working example here? I'm just worried that is not consistent with our other components and you can't see how it looks unless you scroll all the way down.

Not sure how to solve for this tho, any ideas?

Reply:
I wasn't sure either. This was the best I could think of at the time. I'm open to suggestions.